src/webrtc/stats/trackStatsReporter.ts

```diff
@@ -137,26 +137,37 @@ export class TrackStatsReporter {
     }
 
     public static buildTrackSummary(trackStatsList: MediaTrackStats[]): {
-        audioTrackSummary: TrackSummary;
+        audioTrackSummary: AudioTrackSummary;
         videoTrackSummary: TrackSummary;
     } {
-        const audioTrackSummary = { count: 0, muted: 0, maxJitter: 0, maxPacketLoss: 0 };
-        const videoTrackSummary = { count: 0, muted: 0, maxJitter: 0, maxPacketLoss: 0 };
-        trackStatsList
-            .filter((t) => t.getType() === "remote")
-            .forEach((stats) => {
-                const trackSummary = stats.kind === "video" ? videoTrackSummary : audioTrackSummary;
-                trackSummary.count++;
-                if (stats.alive && stats.muted) {
-                    trackSummary.muted++;
-                }
-                if (trackSummary.maxJitter < stats.getJitter()) {
-                    trackSummary.maxJitter = stats.getJitter();
-                }
-                if (trackSummary.maxPacketLoss < stats.getLoss().packetsLost) {
-                    trackSummary.maxPacketLoss = stats.getLoss().packetsLost;
-                }
-            });
+        const videoTrackSummary: TrackSummary = { count: 0, muted: 0, maxJitter: 0, maxPacketLoss: 0 };
+        const audioTrackSummary: AudioTrackSummary = {
+            count: 0,
+            muted: 0,
+            maxJitter: 0,
+            maxPacketLoss: 0,
+            concealedAudioRatio: 0,
+        };
+
+        const remoteTrackList = trackStatsList.filter((t) => t.getType() === "remote");
+        remoteTrackList.forEach((stats) => {
+            const trackSummary = stats.kind === "video" ? videoTrackSummary : audioTrackSummary;
+            trackSummary.count++;
+            if (stats.alive && stats.muted) {
+                trackSummary.muted++;
+            }
+            if (trackSummary.maxJitter < stats.getJitter()) {
+                trackSummary.maxJitter = stats.getJitter();
+            }
+            if (trackSummary.maxPacketLoss < stats.getLoss().packetsLost) {
+                trackSummary.maxPacketLoss = stats.getLoss().packetsLost;
+            }
+        });
+
+        const audioTrackList = remoteTrackList.filter((t) => t.kind === "audio");
+        audioTrackList.forEach((stats) => {
+            audioTrackSummary.concealedAudioRatio += stats.getAudioConcealment()?.ratio / audioTrackList.length;
```

Comment:
I would do this calculation in: https://github.com/matrix-org/matrix-js-sdk/blob/fb3bdc39ecfdb12c0d6c3a3dd6e346d771517fde/src/webrtc/stats/summaryStatsReporter.ts#LL109C8-L109C8

like:

```typescript
summaryCounter.concealedAudio += (stats.audioTrackSummary.count > 0 ? stats.audioTrackSummary.concealedAudioRatio/stats.audioTrackSummary.count : 0)
```

Here i only wold sum the `stats.getAudioConcealment()?.ratio` 

Maybe with my first comment makes also more sense.